model/database/cosmosdb-metrics.yaml

```diff
@@ -0,0 +1,28 @@
+groups:
+  - id: metric.db.client.cosmosdb.operation.request_charge
+    type: metric
+    metric_name: db.client.cosmosdb.operation.request_charge
+    brief: "[Request charge](https://learn.microsoft.com/azure/cosmos-db/request-units) consumed by the operation"
+    instrument: histogram
+    unit: "{request_unit}"
+    stability: experimental
+    extends: attributes.db.cosmosdb.minimal
+
+  - id: metric.db.client.cosmosdb.active_instance.count
+    type: metric
+    metric_name: db.client.cosmosdb.client.active_instance.count
```

Comment:
do you think we can avoid the second `.client.` namespace somehow? e.g. `db.client.cosmosdb.active_client.count` or something else

Reply:
oh yes...thats what it is, sorry for the typo...